.drone.yml

```diff
@@ -153,16 +149,16 @@ platform:
   arch: amd64
 
 steps:
-  - name: package-windows-images
-    image: rancher/dapper:v0.5.5
-    commands:
-      - docker pull --quiet rancher/hardened-build-base:v1.16.4b7
-      - dapper -f Dockerfile --target dapper make package-windows-images
-    when:
-      event:
-        - tag
-      instance:
-        - drone-publish.rancher.io
+- name: package-windows-images
```

Comment:
Keep the whitespace please, for consistency with the rest of the file.

Reply:
fixed